include/arch/arc/v2/linker.ld

```diff
@@ -220,5 +221,11 @@ SECTIONS {
 		KEEP(*(.gnu.attributes))
 	}
 
-	/DISCARD/ : { *(.note.GNU-stack) }
+	/DISCARD/ : {
```

Comment:
Even though that clean-up shouldn't hurt, but does it have anything to do with TLS really?

Reply:
The `.got*` sections are generated when there are `__thread` variables. So they need to be discarded.

Reply:
Makes sense then, didn't think about it.